synapse/handlers/room.py

```diff
@@ -375,12 +375,15 @@ def get_event_context(self, user, room_id, event_id, limit, is_guest):
 
         now_token = yield self.hs.get_event_sources().get_current_token()
 
+        users = yield self.store.get_users_in_room(room_id)
+        is_peeking = user.to_string() not in users
+
         def filter_evts(events):
             return filter_events_for_client(
                 self.store,
                 user.to_string(),
                 events,
-                is_peeking=is_guest
+                is_peeking=is_peeking
```

Comment:
Do we still need the `is_guest` parameter in `get_event_context`? If you are leaving it could you add some docstring to indicate it is unused?